src/Configuration/Config.AzureKeyVault/src/AzureKeyVaultConfigurationExtensions.cs

```diff
@@ -164,12 +176,16 @@ private static async Task<string> GetTokenFromClientCertificate(string authority
         /// <param name="vault">Azure KeyVault uri.</param>
         /// <param name="client">The <see cref="KeyVaultClient"/> to use for retrieving values.</param>
         /// <param name="manager">The <see cref="IKeyVaultSecretManager"/> instance used to control secret loading.</param>
+        /// <param name="reloadOnChange">Whether the configuration should be reloaded if the Azure KeyVault changes.</param>
+        /// <param name="reloadPollDelay">Number of milliseconds to wait inbetween each attempt at polling the Azure KeyVault for changes. Default is 10000 ms.</param>
         /// <returns>The <see cref="IConfigurationBuilder"/>.</returns>
         public static IConfigurationBuilder AddAzureKeyVault(
             this IConfigurationBuilder configurationBuilder,
             string vault,
             KeyVaultClient client,
-            IKeyVaultSecretManager manager)
+            IKeyVaultSecretManager manager,
+            bool reloadOnChange = false,
+            int reloadPollDelay = 10000)
```

Comment:
Take TimeSpan instead of int. I don't think we need two arguments here. `TimeSpan.Zero` should be enough to signal that no reload is required.

Reply:
Thank you. Do we need to update any documents anywhere to specify what TimeSpan.Zero represents? Or updating the method docs would be sufficient?

Reply:
Nullable TimeSpan would be preferable to having zero as a special value.